pages/api.md

```diff
@@ -27,8 +28,17 @@ The following regex matches imports that are *not* Accumulo public API. This
 regex can be used with [RegexpSingleline] to automatically find suspicious
 imports in a project using Accumulo.
 
+For 1.9 and earlier:
+
+```regex
+import\s+org\.apache\.accumulo\.(.*\.(impl|thrift|crypto)\..*|(?!(core\.(client|data|security)|minicluster)\.).*)
 ```
-import\s+org\.apache\.accumulo\.(.*\.(impl|thrift|crypto)\..*|(?!core|minicluster).*|core\.(?!client|data|security).*)
+
+For 2.0 and later, this can be simplified, because sub-packages not intended
+for public API were relocated, and also altered to include the new MapReduce module:
+
+```regex
+import\s+org\.apache\.accumulo\.(?!(core\.(client|data|security)|minicluster|hadoop)\.).*
```

Comment:
Did you test this regex?  I wrote a standalone java program to test the exisitng regex with different strings.

Reply:
I did test it, with https://www.regextester.com, with both the JS engine and the PCRE one. These regexes don't employ anything that would differ substantially from Java's implementation, but I will test it on Java to be sure, before merging.

Reply:
Yep, tested with Java. Works the same as JavaScript and PCRE.